File: StylesAndThemes/app/src/main/java/com/example/hellokitty/MainActivity.java
package com.example.hellokitty;

import static com.example.hellokitty.R.id.editText;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView mHelloTextView;
    private EditText mNameEditText;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mHelloTextView = findViewById(R.id.textView);
        mNameEditText = findViewById(R.id.editText);
    }

    public void onClick(View view) {
        Intent intent = new Intent(MainActivity.this, AboutActivity.class);
        startActivity(intent);
    }
    public void onClick2(View view) {
        Intent intent = new Intent(MainActivity.this, BirthdayActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/hellokitty/MainActivity.kt
package com.example.hellokitty

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageButton
import android.widget.TextView
import com.example.hellokitty.R.id.imageButton
import com.example.hellokitty.R.id.textView

class MainActivity : AppCompatActivity() {
    private lateinit var mHelloTextView: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        mHelloTextView = findViewById(textView)
        var imageButton: ImageButton = findViewById(R.id.imageButton)

        imageButton.setOnClickListener {
            mHelloTextView.setText("aboba")
        }

    }
}